src/Compute/Compute/Strategies/ComputeRp/VirtualMachineStrategy.cs

```diff
@@ -71,6 +71,10 @@ static class VirtualMachineStrategy
             int? vCPUsAvailable = null,
             int? vCPUsPerCore = null,
             string imageReferenceId = null,
+            bool? enableVtpm = null,
+            bool? enableSecureBoot = null,
+            string securityType = null,
+            Dictionary<string, List<string>> auxAuthHeader = null
```

Comment:
you are missing a comma at the end of this line.

Reply:
which is causing the build error.

Reply:
Done thanks